README.md

```diff
@@ -5,7 +5,10 @@ In this repository, some algorithms are implemented in go language.
 
 GoDoc link: [ed](http://godoc.org/github.com/daviddengcn/go-algs/ed) [maxflow](http://godoc.org/github.com/daviddengcn/go-algs/maxflow)
 
-maxflow [![GoSearch](http://go-search.org/badge?id=github.com%2Fdaviddengcn%2Fgo-algs%2Fmaxflow)](http://go-search.org/view?id=github.com%2Fdaviddengcn%2Fgo-algs%2Fmaxflow)
+### About Max-flow problem:
+A flow network is represented in a directed acyclic graph(DAG). Each edge has a nonnegative capacity, to which the flow is limited. There are a source node s and a sink node t. s has no incoming edges, and t has no outgoing edges. All other nodes are internal nodes, in which the amount of incoming flow must equal to the amount of ougoing flow. The goal of the max-flow problem is, given a flow network, to find a flow of maximum value.
```

Comment:
Thanks for your interests on this libary.

For the introduction, another simpler way is to provide a like to wikipedia:

https://en.wikipedia.org/wiki/Maximum_flow_problem

What do you think?

Reply:
Yes, it definitely works to provide a Wikipedia link. I just thought it could use a quick summary without having a reader to go to a link. Thanks for your suggestion!

Reply:
I'll accept it first and may optimize it later. Thanks for the contribution!